live-build/misc/upgrade-scripts/verify

```diff
@@ -20,18 +20,7 @@
 IMAGE_PATH=$(get_image_path)
 [[ -n "$IMAGE_PATH" ]] || die "failed to determine image path"
 
-IMAGE_VERSION=$(get_image_version)
-[[ -n "$IMAGE_VERSION" ]] || die "failed to determine image version"
-
-function usage() {
-	echo "$(basename "$0"): $*" >&2
-	echo "Usage: $(basename "$0") -v <version>"
-	exit 2
-}
-
-function report_progress_inc() {
-	echo "Progress increment: $(date +%T:%N%z), $1, $2"
```

Comment:
@dtzeng Also note, this change is removing the progress reporting output lines. I'm not convinced the current reporting is sufficient for what we'll eventually need on Linux, so I've opted to just remove it altogether for now, and plan to circle back to this problem later.

For example, there's certain commands that can take a long time to complete (e.g. `apt-get install`, `debootstrap`), so I'm not sure if the current approach of simply emitting progress lines in between each of the commands we run will be enough.

Let me know if you're OK removing these for now, or if you'd rather me add them back (even if they may not be sufficient when this is all done).

Reply:
I think we should actually put these back.  We don't really have any short-term plans to re-vamp any of the progress checks (at least for L.0).  And having some progress is better than nothing

Otherwise, this review looks good to me

Reply:
OK. I'll add it back.